### DISCUSSION TITLE
disable key on certain websites

### DISCUSSION THREAD

**Discussion by onguarde** [brookhong/Surfingkeys#1691] [Q&A] [answered]
I want to disable the '/' key on google, so it will default to going to the search bar on the page, instead of Surfingkey's find. 

I tried the following it doesn't work,

```
if ( document.origin === "https://www.google.com/" ) {
    unmap('/');
}
```

**Comment by onguarde** [comment, marked as answer]
This works,

```
if( /^(.*\.)?google\./.test(window.location.host) ){
    unmap('/');
}
```

Reference,
https://stackoverflow.com/questions/2522532/how-can-i-use-javascript-to-match-a-string-inside-the-current-url-of-the-window

**Comment by userrand** [comment]
there seems to be code for this with a commented example in the code here https://github.com/brookhong/Surfingkeys/blob/6d0ee1dfda4e5231920bda0a20996134d1e0ba29/src/content_scripts/common/api.js#LL163C9-L163C9

here is another example (note the absence of quotes in the domain name) `api.unmap("j",/google.com/);`
